daml/Common.daml

```diff
@@ -58,10 +63,12 @@ onboardCommon = do
   matchingServiceCid <- submit exchange do exerciseCmd matchingServiceOfferCid Matching.Accept
   settlementServiceOfferCid <- submit operator do exerciseCmd operatorServiceCid Operator.OfferSettlementService with provider = exchange
   settlementServiceCid <- submit exchange do exerciseCmd settlementServiceOfferCid Settlement.Accept
+  auctioneerRoleOfferCid <- submit operator do exerciseCmd operatorServiceCid Operator.OfferAuctioneerRole with provider = auctionAgent
+  auctioneerRoleCid <- submit auctionAgent do exerciseCmd auctioneerRoleOfferCid Auctioneer.Accept
   pure CommonContracts with ..
 
-onboardCustomer : CommonContracts -> Text -> Asset -> Script CustomerContracts
-onboardCustomer CommonContracts{..} party asset = do
+onboardCustomer : CommonContracts -> Text -> Asset -> Party -> Script CustomerContracts
+onboardCustomer CommonContracts{..} party asset accountNominee = do
```

Comment:
You mentioned this during the call. Can you explain why we need this? I thought we'd agreed that for the time being we wouldn't have a separate party for the auction, we could use the custodian as he's authorized on all the accounts?

Reply:
explained in Slack